Willning/DLOplanTool
# DLO-19

### **Related Issue/Keyword:**
DLO-19
### **Description:**
Added Front End popup for creating the new project.

### **Testing:**
- Topic and title entry are both able to be filled out
- Create button is not clickable unless have title and topic 
- Image uploading displays the uploaded image
- Image upload doesn't accept non-images

**Steps for manual testing:**

1. Click new project opens popup
2. Enter title and topic
3. Drag picture
4. Create should push show info on the stub project page. 

### **Checklist:**
